dev/main.sh

```diff
@@ -58,7 +103,7 @@ for tenant in "${tenants[@]}"; do
         echo "🐈Provisioning missing tenant: $tenant"
         baseURL="http://$tenant.stellar.local:8000"
         sdpUIBaseURL="http://$tenant.stellar.local:3000"
-        ownerEmail="owner@$tenant.local"
+        ownerEmail="init_owner@$tenant.org"
```

Comment:
The reason we were using `@tenant.local` for emails is to not reference an existing domain.